packages/headless/src/controllers/knowledge/generated-answer/headless-answerapi-generated-answer.ts

```diff
@@ -163,6 +164,9 @@ export function buildAnswerApiGeneratedAnswer(
       return {
         ...getState().generatedAnswer,
         answer: answerApiState?.answer,
+        citations: filterOutDuplicatedCitations(
+          answerApiState?.citations ?? []
+        ),
```

Comment:
```suggestion
        citations: answerApiState?.citations?.length
          ? filterOutDuplicatedCitations(answerApiState.citations)
          : [],
        ),
```

Just a suggestion to avoid calling the function on an empty array, maybe im overthinking this though x)

Reply:
good with me!